packages/vscode-extension/src/handlers.ts

```diff
@@ -925,7 +925,10 @@ export async function openHelpFeedbackLinkHandler(args: any[]): Promise<boolean>
 }
 export async function openWelcomeHandler(args?: any[]) {
   ExtTelemetry.sendTelemetryEvent(TelemetryEvent.QuickStart, getTriggerFromProperty(args));
-  WebviewPanel.createOrShow(PanelType.QuickStart);
+  vscode.commands.executeCommand(
```

Comment:
So the current walkthrough will replace the quick start instead of existing at the same time?